MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -8395,8 +8395,8 @@ public boolean checkRetirementDefections() {
 
     public boolean checkYearlyRetirements() {
         if (getCampaignOptions().getUseAtB()
-                && Utilities.getDaysBetween(getRetirementDefectionTracker()
-                        .getLastRetirementRoll().getTime(), getDate()) == 365) {
+                && (Math.toIntExact(ChronoUnit.YEARS.between(getRetirementDefectionTracker()
+                .getLastRetirementRoll(), getLocalDate())) == 1)) {
```

Comment:
`ChronoUnit.YEARS.between` returns the whole number of years between two dates. This isn't the same logic as before (but perhaps *roughly* the same logic).

Reply:
That is correct, but the rules do say yearly so this is proper